frontend/resourceadm/pages/DeployResourcePage/DeployResourcePage.tsx

```diff
@@ -119,10 +86,10 @@ export const DeployResourcePage = ({ navigateToPageWithError }: Props) => {
   const getStatusCardType = (): 'danger' | 'success' => {
     // TODO - Add check for if version is correct
     if (
-      hasResourceError ||
+      validateResourceData.status !== 200 ||
```

Comment:
As mentioned above, the status checks is on the actual object returned from backend. It can be 400 even if it is in the success (200) part of the API Call.